.gitignore

```diff
@@ -8,3 +8,4 @@ gdp-src-build/downloads
 gdp-src-build/sstate-cache
 gdp-src-build/tmp/
 gdp-src-build/logs
+gdp-src-build/tmp-glibc
```

Comment:
Is this a generated directory?  I have not seen it before but if it is something that is created during the build I assume it should be in .gitignore and not commited?